hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2316,9 +2316,7 @@ func reconcileCAPIManagerDeployment(deployment *appsv1.Deployment, hc *hyperv1.H
 	}
 
 	hyperutil.SetColocation(hc, deployment)
-	// TODO (alberto): Reconsider enable this back when we face a real need
```

Comment:
what has changed? Do we have any new feedback where this was needed? If so, is there an underlying issue ticket where we are tracking the underlying root problem?

Reply:
1) IBM has operational requirements that a restart of all components can be initiated on demand and is fully automated

2) CA rotation is failing since the ignition-server is not reloading certs when they are updated. After the old CA is revoked all ignition boots fail because of a X509 untrusted error.

Reply:
Tracking here now https://issues.redhat.com/browse/HOSTEDCP-307